### DISCUSSION TITLE
Change cloudflare domain link

### DISCUSSION THREAD

**Discussion by PhPersson** [homeassistant-apps/app-cloudflared#486] [Ideas]
The current link to cloudflare domain service does not work [Get a Domain name](https://github.com/brenner-tobias/addon-cloudflared/wiki/How-tos#get-a-domain-name)

Could mabye instead use a link to the developers docs? [Developers docs registrar domain](https://developers.cloudflare.com/registrar/get-started/register-domain/)

**Comment by brenner-tobias** [comment]
Thanks for raising @PhPersson. Unfortunately, I am not able to find the link you are reffering to. Can you please be more specific about the link / add more information for me?

**Reply by PhPersson** [reply]
Sorry for being unclear. 
current link https://www.cloudflare.com/products/registrar/ seems to not work in the readme. 
Maybe it could be changed to https://developers.cloudflare.com/registrar/get-started/register-domain/ instead, however I don’t know how valuable this link is for people without proper knowledge about cloudflare and their DNS works.

**Reply by brenner-tobias** [reply]
Thanks. The Link was working fine for me, though it was forwarding to https://www.cloudflare.com/en-gb/products/registrar/
I did change the link to the forward, so everything should work now for you as well.
Can you please check and let me know?

**Reply by PhPersson** [reply]
The link u provided seems to work fine! 
strange it did not forward for me.
